### ISSUE TITLE
JavaFX webkit error with version 610.1

### ISSUE THREAD

**Issue by buedi** [ojdkbuild/ojdkbuild#131]
A significant animation bug could be fixed with the update from 610.1 to 610.2

https://bugs-stage.openjdk.java.net/browse/JDK-8259635

This error stops CSS-based animations, for example in SVG files.
Is it possible to update the WebKit for the next version like in the java8 openjdk?

The functionality can be tested with the following .svg graphic file. 

The Animation (rectangleColorFade) is working with the webkit 608 (replaces in the ojdkbuild project 6 moth ago) with the version 610.1 (not working). I tested the animation the actual oracle JDK (jdk1.8.0_291) and the animation is running again with webKit version 610.2

```
<?xml version="1.0" encoding="UTF-8" standalone="no"?>
<svg xmlns:dc="http://purl.org/dc/elements/1.1/" xmlns:cc="http://creativecommons.org/ns#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:svg="http://www.w3.org/2000/svg" xmlns="http://www.w3.org/2000/svg" xmlns:sodipodi="http://sodipodi.sourceforge.net/DTD/sodipodi-0.dtd" xmlns:inkscape="http://www.inkscape.org/namespaces/inkscape" width="744.09448" height="524.40942" id="svg2" inkscape:version="1.0.2-2 (e86c870879, 2021-01-15)" sodipodi:docname="drawing.svg">
  <defs id="defs4"></defs>
  <style type="text/css" id="style2996"><![CDATA[
	.rectangleColorFade {
		animation-name: colorChange;
		animation-duration: 4s;
		animation-iteration-count: infinite;
	}

   @keyframes colorChange {
		0%  {fill: red;}
		25% {fill: yellow;}
		50% {fill: blue;}
		100% {fill: green;}
	}
  ]]></style>
  <metadata id="metadata7"></metadata>
  <rect class="rectangleColorFade" id="rectangle" width="250.0" height="180.0" x="260.0" y="130.0"></rect>
</svg>


```

Full maven project:
[Maven_project_WebViewTest.zip](https://github.com/ojdkbuild/ojdkbuild/files/6376574/Maven_project_WebViewTest.zip)

![image](https://user-images.githubusercontent.com/5691507/116073262-74340600-a690-11eb-997c-2206501d4e60.png)
Working with oracle JDK (jdk1.8.0_291) 

![image](https://user-images.githubusercontent.com/5691507/116073507-cb39db00-a690-11eb-847e-ef9c70900ff6.png)
Working with ojdkbuild (jdk1.8.0_265) 

![image](https://user-images.githubusercontent.com/5691507/116073665-01775a80-a691-11eb-8751-bb23ca834a74.png)
Not working with ojdkbuild (jdk1.8.0_282)

**Comment by buedi** [comment]
The Problem is solved with ojdkbuild 1.8.0.292.
Thanks for this update!

![image](https://user-images.githubusercontent.com/5691507/116868137-87098600-ac0e-11eb-945b-ff3319fbfd0d.png)

**Comment by ojdkbuild** [comment]
Thanks for the report! Sorry for the delayed response, it was intended to update OpenJFX for this release, but I haven't yet got a chance to check the results. Thanks for verifying them!
